stripe.go

```diff
@@ -292,8 +304,15 @@ func (s *BackendImplementation) Do(req *http.Request, body *bytes.Buffer, v inte
 		s.Logger.Printf("Requesting %v %v%v\n", req.Method, req.URL.Host, req.URL.Path)
 	}
 
+	if EnableTelemetry && s.lastRequestMetrics != nil {
+		metricsJSON, _ := json.Marshal(s.lastRequestMetrics)
+		payload := fmt.Sprintf(`{"last_request_metrics":%s}`, metricsJSON)
```

Comment:
Just for cleanliness, could you create a wrapper struct for this one instead of manually assembling the JSON string (maybe `requestTelemetry`)?